be/src/runtime/bufferpool/suballocator.h

```diff
@@ -128,6 +129,10 @@ class Suballocator {
   // if list is empty.
   std::unique_ptr<Suballocation> PopFreeListHead(int list_idx);
 
+  // Get the allocation at the head of the free list at index 'list_idx'. Return nullptr
+  // if list is empty.
+  bool CheckFreeListHead(int list_idx) const;
+
```

Comment:
Modify annotation， the annotation is not consistent with function.

Reply:
ok，I will change the name